src/components/Dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -0,0 +1,65 @@
+import { useAppList } from 'src/routes/safe/components/Apps/hooks/appList/useAppList'
+import { Text } from '@gnosis.pm/safe-react-components'
+import { Link } from 'react-router-dom'
+import { getSafeAppUrl, SafeRouteParams } from 'src/routes/routes'
+import { useSelector } from 'react-redux'
+import { currentSafe } from 'src/logic/safe/store/selectors'
+import { getShortName } from 'src/config'
+import { ReactElement, useMemo } from 'react'
+import Row from 'src/components/layout/Row'
+import Col from 'src/components/layout/Col'
+import styled from 'styled-components'
+
+const StyledImage = styled.img`
+  max-width: 64px;
+  max-height: 64px;
+`
+
+const StyledLink = styled(Link)`
+  margin-top: 10px;
+  text-decoration: none;
+`
+
+const StyledRow = styled(Row)`
+  gap: 24px;
+  flex-wrap: inherit;
+`
+
+// TODO: Replace once tags are available
+const featuredAppIds = ['29', '11']
```

Comment:
Tags are available on staging. Can we use them here?

Reply:
Good point, I switched over to filter by tags and removed the hard-coded ids. We still need to update the type in the gateway sdk. For the time being I extended the type here.